<?xml version="1.0" encoding="UTF-8"?>


<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>601</width>
                <height>216</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Auto Setup</string>
        </property>
        <property name="styleSheet">
            <string notr="true">font: 11pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label_2">
                    <property name="styleSheet">
                        <string notr="true">background-color: rgb(175, 217, 248)</string>
                    </property>
                    <property name="text">
                        <string/>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <spacer name="horizontalSpacer_5">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="font">
                                <font>
                                    <family>.AppleSystemUIFont</family>
                                    <pointsize>11</pointsize>
                                    <weight>50</weight>
                                    <italic>false</italic>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string>Machine Amplitude:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="machine_readback_label">
                            <property name="font">
                                <font>
                                    <family>.AppleSystemUIFont</family>
                                    <pointsize>11</pointsize>
                                    <weight>50</weight>
                                    <italic>false</italic>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string>READBACK</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_6">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="machine_setup_button">
                            <property name="text">
                                <string>Set Up Machine</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="machine_shutdown_button">
                            <property name="text">
                                <string>Shut Down Machine</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="machine_abort_button">
                            <property name="text">
                                <string>Abort Machine</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                        <spacer name="horizontalSpacer_7">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_3">
                            <property name="font">
                                <font>
                                    <family>.AppleSystemUIFont</family>
                                    <pointsize>11</pointsize>
                                    <weight>50</weight>
                                    <italic>false</italic>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string>Expert Options:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="ssa_cal_checkbox">
                            <property name="toolTip">
                                <string>Leave all checked if unsure</string>
                            </property>
                            <property name="text">
                                <string>SSA Calibration</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="autotune_checkbox">
                            <property name="toolTip">
                                <string>Leave all checked if unsure</string>
                            </property>
                            <property name="text">
                                <string>Auto Tune</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="cav_char_checkbox">
                            <property name="toolTip">
                                <string>Leave all checked if unsure</string>
                            </property>
                            <property name="text">
                                <string>Cavity Characterization</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="rf_ramp_checkbox">
                            <property name="toolTip">
                                <string>Leave all checked if unsure</string>
                            </property>
                            <property name="text">
                                <string>RF Ramp</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_8">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QTabWidget" name="tabWidget_linac"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
